gravitee-apim-integration-tests/src/test/java/io/gravitee/apim/integration/tests/secrets/conf/KubernetesSecretProviderIntegrationTest.java

```diff
@@ -532,9 +534,20 @@ void createSecrets() {
 
         @Override
         public void setupAdditionalProperties(GatewayConfigurationBuilder configurationBuilder) {
-            configurationBuilder.setYamlProperty("missing", "secret://kubernetes/test:pass");
-            configurationBuilder.setYamlProperty("missing2", "secret://kubernetes/test:pass?namespace=test");
-            configurationBuilder.setYamlProperty("no_plugin", "secret://foo/test:pass");
+            // We can't add invalid secret here unless the gateway will fail to start.
+        }
```

Comment:
should we remove this abstraction and do all the property modification in the beforeAll (for all the tests) ?

Reply:
This test requires setting up additional property differently than usual so I would prefer to keep the existing way to add additional yaml property as it is (which is a clean way to do IMHO).